tasks/Investment Game.md

```diff
@@ -0,0 +1,32 @@
+# Summary
+> The Investment Game, AKA the Trust Game, is an economic game designed to demonstrate that trust is intrinsic to economic transactions. Divided into two stages, the first has individuals send money to a person in another room, that money is then tripled. In the second stage, the other person sends money back to the original individual.
+
+# References
+> Berg, J., Dickhaut, J., & McCabe, K. (1995). Trust, Reciprocity, and Social History. Games and Economic Behavior, 10(1), 122–142.
+> https://econweb.ucsd.edu/~jandreon/Econ264/papers/Berg%20et%20al%20GEB%201995.pdf
+
+# Stimuli
+## The visual components
+> The experiment took place in person. Subjects were in each condition were in separate rooms. The subjects in room A were handed a large manilla envelope with a smaller envelop to place money in and a key corresponding to the room they were in. Subjects were then instructed to go behind a privacy partition and place the amount of money they wanted to “invest” in the envelope. The subject then places the large opaque envelope into the return box.
+
+
+# Procedure
+## Steps
+Room A:
+1 Given 10 dollar ‘Showing up fee’
+2 Given two envelopes, one larger one with a key, and a smaller one with a corresponding letter
+3 Subjects go behind privacy partition and place money to invested in the lettered envelope
+4 Subjects then place envelope in mailbox
+Room B: 
+1 Participants receive triple the amount given by participant in room A
+Repeat steps 2-4 from room A
+
+## Rolls 
+> Subjects are assigned to either room A or B. Subjects in room A are given a 10 dollar ‘showing up fee’ and have to decide how much to send to subjects in room B. Room B participants receive tripled money and are asked to decide how much to send back to those in room A.
+
+# Criteria
+## Performance calculation
+> There is no correct answer. The amount of money being sent back and forth is the only thing being measured.
```

Comment:
I would note that ... -> A metric evaluated by the source paper was the mean and deviation of the amount of money sent from both rooms A and B in the 2 versions of the experiment (supplying the history chart for the subjetcs in the beginning vs not supplying it)